.github/workflows/_stage_publish.yml

```diff
@@ -107,3 +107,5 @@ jobs:
         ref: main
         repository: ${{ inputs.version_dev != '' && 'envoyproxy/envoy-website' || 'envoyproxy/archive' }}
         workflow: envoy-sync.yaml
+        inputs: |
+          commit_sha: ${{ inputs.version_dev != '' && github.sha || '' }}
```

Comment:
can you add a comment in this file where this gets used?  I was unable to find github docs or our repo

Reply:
sure ill add a comment - fwiw the relevant info is in the lines above ...

```yaml
        repository: ${{ inputs.version_dev != '' && 'envoyproxy/envoy-website' || 'envoyproxy/archive' }}
        workflow: envoy-sync.yaml
```

Reply:
re documenting the process - im still finalizing a few things - but have a PR in the website repo that will make things a bit clearer when im done with it - probs we can link  to that somewhere in envoy repo